inspirehep/modules/migrator/legacy/audit.py

```diff
@@ -24,3 +24,33 @@
 """INSPIRE migrator load/dump audit."""
 
 from __future__ import absolute_import, print_function
+
+import json
+
+from sqlalchemy.orm.exc import NoResultFound  # , MultipleResultsFound
+
+
+def date_handler(obj):
+    return obj.isoformat() if hasattr(obj, 'isoformat') else obj
```

Comment:
What is date_handler? Is it returning a date? If yes, it should not return `obj`.